### PR TITLE
Added force remove application cmdlet in case of failure of remove application cmdlet

### PR DESCRIPTION
**Pull request by kuvinodms**

### CODE REVIEW

File: Tasks/ServiceFabricDeploy/ServiceFabricSDK/Publish-NewServiceFabricApplication.ps1 [hunk 1]
```diff
@@ -214,7 +214,7 @@
 			    }
 				catch [System.TimeoutException]
 				{
-					# Catch operation timeout and continue with force remove replica.
+					$app | Remove-ServiceFabricApplication -ForceRemove
```
Comment (NCarlsonMSFT) [inline Tasks/ServiceFabricDeploy/ServiceFabricSDK/Publish-NewServiceFabricApplication.ps1]: I think you still need to add -Force to bypass a confirmation prompt: [docs](https://docs.microsoft.com/en-us/powershell/module/servicefabric/remove-servicefabricservice?view=azureservicefabricps#optional-parameters)

File: Tasks/ServiceFabricDeploy/ServiceFabricSDK/Publish-NewServiceFabricApplication.ps1 [hunk 2]
```diff
@@ -214,7 +214,7 @@
...
 				}
 
                 foreach ($node in Get-ServiceFabricNode)
```
Comment (bishal-pdMSFT) [inline Tasks/ServiceFabricDeploy/ServiceFabricSDK/Publish-NewServiceFabricApplication.ps1]: why do we still need to remove replica from each node?

Comment (kuvinodms) [reply Tasks/ServiceFabricDeploy/ServiceFabricSDK/Publish-NewServiceFabricApplication.ps1]: Updated

File: Tasks/ServiceFabricDeploy/ServiceFabricSDK/Publish-NewServiceFabricApplication.ps1 (hunk 1 above)
Comment (kuvinodms) [reply Tasks/ServiceFabricDeploy/ServiceFabricSDK/Publish-NewServiceFabricApplication.ps1]: Correct. Missed while committing. Modified and tested the statement now.

**Pull request merged**